src/main/java/org/terasology/module/behaviors/plugin/CompositeMovementPlugin.java

```diff
@@ -22,11 +22,6 @@
 
     private final List<MovementPlugin> plugins;
 
-    public CompositeMovementPlugin(WorldProvider world, Time time, MovementPlugin... plugins) {
```

Comment:
One place where we might want to use the constructor with varargs is when creating a composite plugin by hand in tests. 

If this is not used anywhere right now, I think we can safely remove it, and add it back in case we notice that it would help us write tests.

Reply:
According to Intellij it's not used anywhere right now.